framework/src/modules/token/schemas.ts

```diff
@@ -226,29 +227,27 @@ export const crossChainTransferMessageParams = {
 	properties: {
 		tokenID: {
 			dataType: 'bytes',
+			length: TOKEN_ID_LENGTH,
```

Comment:
`length` actually doesn't exist on schema, so in the implementation `minLength` and `maxLength` are correct